scripts/create-hashes/index.ts

```diff
@@ -0,0 +1,73 @@
+import crypto from "node:crypto";
+import fs from "node:fs";
+import { writeFile } from "node:fs/promises";
+import path from "node:path";
+import readline from "node:readline";
+
+interface InputLine {
+  location?: string;
+  base?: string;
+}
+
+interface HashRecord {
+  file: string;
+  sha384: string;
+  sha512: string;
+}
+
+main().catch((error) => {
+  console.error(error);
+
+  process.exit(1);
+});
+
+function createFileHash(algorithm: string, filename: string, base: string) {
+  const fileBuffer = fs.readFileSync(path.join(base, filename));
+  const hash = crypto.createHash(algorithm);
+
+  hash.update(fileBuffer);
+  return btoa(hash.digest("binary"));
+}
+
+function includeFile(filename: string) {
+  return [".scss", "readme.md", "README.md", "CHANGELOG.md", "DISCLAIMER.txt", ".d.ts"].every(
+    (exclude) => !filename.endsWith(exclude),
+  );
```

Comment:
Tsja, ik hoop niet dat je mij een zeur vindt... Maar dit is eigenlijk ook niet ideaal: Voor elke keer dat `includeFile()` wordt aangeroepen wordt deze array gemaakt.

Het is netter om een array als een const buiten de functie te zetten.

Reply:
Gedaan.

Reply:
Thanks goos